client/src/admin/AlertForm.js

```diff
@@ -3,28 +3,37 @@ import { Field, reduxForm } from 'redux-form';
 import alertValidator from './alertValidator';
 
 const AlertForm = ({ featurePicked, handleSubmit, submitting, valid }) => (
-  <form className="usa-form" onSubmit={handleSubmit}>
-    <fieldset>
-      <div>
-        <label htmlFor="alertMessage">Alert Message</label>
-        <Field
-          name="alertMessage"
-          component="textarea"
-          placeholder="Remember to include a link to more information"
-          type="text"
-        />
-      </div>
-      <div>
-        <Field
-          name="isEmergency"
-          component="input"
-          type="checkbox"
-        />
-        <label htmlFor="isEmergency">Is this an immediate emergency?</label>
-      </div>
-      <input type="submit" disabled={submitting || !(valid && featurePicked)} value="Send Alert!" />
-    </fieldset>
-  </form>
+  <div className="container--span">
+    <form onSubmit={handleSubmit}>
+      <legend className="legend--has-subtitle">
+        <h3>Enter a message describing the event:</h3>
+        <span className="text--subtitle">
+          Make sure to provide a link to allow the reader to get additional information.
```

Comment:
Can you integrate this message with the message I added in the last PR that provides a link to more information if we have it?